packages/core/integration-tests/test/postcss.js

```diff
@@ -291,9 +291,6 @@ describe('postcss', () => {
       {
         name: 'style.css',
         assets: ['style.css'],
-        includedFiles: {
```

Comment:
These aren't attached to asset objects anymore, so there isn't a good way to test this without access to the request graph. These tests weren't really useful anyway since they didn't assert that the cache was invalidated properly, just that the file was listed.

Reply:
I found them useful as a kind of unit test for transformers, as testing this list would be enough under the assumption that core handled it correctly (so that we don't need slow/complicated cache tests for every plugin).

Reply:
Yeah. I think we'll find a way to bring them back. We need a way for the tests to access the internal bundle graph I guess...

Reply:
This kind of felt like testing implementation details anyway. I've been thinking it would be nice to expose the RequestGraph and requests in test utils somehow so transformer plugins could have tests that only run an asset request for instance.